packages/pipeline/src/data_sources/dex_prices/index.ts

```diff
@@ -21,12 +23,21 @@ export interface EdpsExchange {
 // tslint:disable:prefer-function-over-method
 // ^ Keep consistency with other sources and help logical organization
 export class EdpsSource {
+    // rate-limit for all API calls through this class instance
+    private readonly _limiter: Bottleneck;
+    constructor(maxReqsPerSecond: number) {
+        this._limiter = new Bottleneck({
+            minTime: ONE_SECOND / maxReqsPerSecond,
+        });
+    }
+
     /**
      * Call Ethereum DEX Price Service API.
      */
     public async getEdpsAsync(direction: string, symbol: string, amount: number): Promise<EdpsWrapper> {
         const edpsUrl = `${EDPS_BASE_URL}/${direction}?amount=${amount}&symbol=${symbol}&decimals=`;
-        const resp = await fetchAsync(edpsUrl);
+        logUtils.log(`${direction} ${amount} ${symbol}`);
```

Comment:
Did you mean to leave this in?

Reply:
No 🙈- nicely spotted. Thanks! Fixed now.